docs/GHA.md

```diff
@@ -1,60 +1,66 @@
 
-## Partially supported properties
-
-### concurrency
-
-The `concurrency` key within a GitHub Action workflow can be defined at topmost level, and within a single job. The Buildkite Migration tool supports the transition of the `concurrency` key (and `group`/`cancel-in-progress`) in job definitions: and maps to the `concurrency_group` [key](https://buildkite.com/docs/pipelines/controlling-concurrency#concurrency-groups) available within Buildkite. Buildkite also allows a upper limit on how much jobs are created through a single step definition with the `concurrency` key: which is set as `1` by default (there isn't a translatable key within a GitHub Action workflow).
-
-The `cancel-in-progress` Boolean value that can be defined in a `concurrency` hash inside a GitHub Action job workflow maps to the Buildkite pipeline setting of [Cancel Intermediate Builds](https://buildkite.com/docs/pipelines/skipping#cancel-running-intermediate-builds) - which can be set within a pipeline's settings page, or when creating/updating a pipeline via the [REST](https://buildkite.com/docs/apis/rest-api/pipelines#create-a-yaml-pipeline)/[GraphQL](https://buildkite.com/docs/apis/graphql/schemas/mutation/pipelinecreate) APIs.
-
-### defaults
-
-A `defaults` key allows the definition of a hash in which configuration will apply to all of a workflow's jobs. 
-
-While there isn't a direct transition from a GitHub Action workflow to a Buildkite pipeline, Buildkite pipeline definitions allow for the utilisation of [YAML anchors](https://buildkite.com/docs/plugins/using#using-yaml-anchors-with-plugins) in which an anchor can be reused throughout various step definitions within a pipeline.
+## Concurrency (`concurrency`)
 
-Alternatively, the Buildkite platform allows jobs to have default configuration applied to them through the use of [Agent](https://buildkite.com/docs/agent/v3/hooks#agent-lifecycle-hooks) or [Job](https://buildkite.com/docs/agent/v3/hooks#job-lifecycle-hooks) Lifecycle hooks. Agent lifecycle hooks allows the configuration of customized commands on agent start up and shut down. Job Lifecycle hooks allow for the customization of what occurs at each stage of a job's lifecycle - and can be specified and applied at an agent, repository or plugin level.
+| Key | Supported? | Notes |
+| --- | ---------- | ----- |
+| `concurrency` | 🔴 | The Buildkite Migration tool supports the transition of the `concurrency` key (and `group`/`cancel-in-progress`) keys in job definitions that make up a workflow. |
```

Comment:
The icon says no, but the text says yes :thinking:

Reply:
Tweaked the wording - technically we dont @ top level (as GHA allows CGs to be set to all jobs via this key, BK is in the specific command step which we parse 👍)

Reply:
I see what you are trying to say here.